### PR TITLE
feat[FX-4145]: add new artists section in artwork sidebar

### PR DESCRIPTION
**Pull request by mdole**
The type of this PR is: **Feat**

This PR solves [FX-4145]

### Description

With 1 artist
Screen Shot 2022-08-30 at 18 41 07
---
With > 4 artists, not expanded
Screen Shot 2022-08-30 at 18 37 35
---
With > 4 artists, expanded
Screen Shot 2022-08-30 at 18 37 32
---
With cultural maker
Screen Shot 2022-08-30 at 18 38 02

[FX-4145]: https://artsyproduct.atlassian.net/browse/FX-4145?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by damassi** [comment]
Nice working through the palette changes 👌 It is indeed better.

File: src/Apps/Artwork/ArtworkApp.tsx [hunk 1]
```diff
@@ -198,7 +198,7 @@ export const ArtworkApp: React.FC<Props> = props => {
 
         <Column span={4} pt={[0, 2]}>
           {isNewArtworkSidebarEnabled ? (
-            <ArtworkSidebar2 />
+            <ArtworkSidebar2FragmentContainer artwork={artwork} />
```
Comment (gkartalis) [inline src/Apps/Artwork/ArtworkApp.tsx:201]: 🔥

File: src/Apps/Artwork/Components/ArtworkSidebar2/ArtworkSidebar2Artists.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+export const ArtworkSidebar2Artists: React.FC<ArtistsProps> = ({
+  artwork: { artists, culturalMaker },
+}) => {
+  if (!artists) return null
+
+  const showMoreText = `${artists.length - ARTISTS_TO_DISPLAY} more`
+
+  return (
+    <div>
```
Comment (gkartalis) [inline src/Apps/Artwork/Components/ArtworkSidebar2/ArtworkSidebar2Artists.tsx:33]: **question:** is there a reason that you picked a `<div>` here over a `<>` fragment or a `<Box>` ?

**Review by gkartalis — APPROVED** [review]

File: src/Apps/Artwork/Components/ArtworkSidebar2/ArtworkSidebar2Artists.tsx (hunk 2 above)
Comment (mdole) [reply src/Apps/Artwork/Components/ArtworkSidebar2/ArtworkSidebar2Artists.tsx:33]: Not really - it's a personal preference thing. Basically, why use `<>` when it's actually just a `<div>`? AFAIK there's not any advantage to it, it's just saving 3 characters (6 if you count the closing tag) and using components where we could be writing HTML sometimes gets us into trouble 🤷 

But I'm definitely not consistent about it either!

**Pull request merged**